ng/src/web/pages/credentials/dialog.js

```diff
@@ -61,6 +61,25 @@ const type_names = {
   snmp: _('SNMP'),
 };
 
+const DEFAULTS = {
+    allow_insecure: 0,
+    auth_algorithm: SNMP_AUTH_ALGORITHM_SHA1,
+    autogenerate: 0,
+    base: USERNAME_PASSWORD_CREDENTIAL_TYPE,
+    change_community: '0',
+    change_passphrase: '0',
+    change_password: '0',
+    change_privacy_password: '0',
+    comment: '',
+    community: '',
+    credential_login: '',
+    name: _('Unnamed'),
+    passphrase: '',
+    password: '',
+    privacy_algorithm: SNMP_PRIVACY_ALGORITHM_AES,
+    privacy_password: '',
+  };
```

Comment:
Could you change the 1 and 0 values to use YES_VALUE and NO_VALUE from gmp/parser? Also e.g. change_community should be parsed with parse_yesno in the corresponding model class. This seems to be a leftover.

Reply:
Done